### ISSUE TITLE
Can't build on raspberry pi

### ISSUE THREAD

**Issue by jungle-boogie** [jarun/nnn#1047]
#### Environment details (Put `x` in the checkbox along with the information)

- [x] Operating System:
```
$ cat /etc/os-release
PRETTY_NAME="Raspbian GNU/Linux 10 (buster)"
NAME="Raspbian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
```

- [x] Shell: Bash
- [x] Issue exists on `nnn` master

#### Exact steps to reproduce the issue

```
$ make clean && make
rm -f -f nnn nnn-4.0.tar.gz *.sig nnn-static nnn-static-4.0.x86_64.tar.gz nnn-icons-static nnn-icons-static-4.0.x86_64.tar.gz nnn-nerd-static nnn-nerd-static-4.0.x86_64.tar.gz
cc  -std=c11 -Wall -Wextra -Wshadow -O3 -D_DEFAULT_SOURCE -D_XOPEN_SOURCE=600  -o nnn src/nnn.c -lreadline -lncursesw -ltinfo -lpthread
/usr/bin/ld: /tmp/ccHru9ff.o: in function `statusbar':
nnn.c:(.text+0x5868): undefined reference to `__atomic_load_8'
/usr/bin/ld: /tmp/ccHru9ff.o: in function `dentfill.constprop.20':
nnn.c:(.text+0x71ac): undefined reference to `__atomic_fetch_add_8'
/usr/bin/ld: nnn.c:(.text+0x72d4): undefined reference to `__atomic_store_8'
/usr/bin/ld: nnn.c:(.text+0x7664): undefined reference to `__atomic_fetch_add_8'
/usr/bin/ld: nnn.c:(.text+0x7750): undefined reference to `__atomic_fetch_add_8'
collect2: error: ld returned 1 exit status
make: *** [Makefile:147: nnn] Error 1
```

```
$ cc -v
Using built-in specs.
COLLECT_GCC=cc
COLLECT_LTO_WRAPPER=/usr/lib/gcc/arm-linux-gnueabihf/8/lto-wrapper
Target: arm-linux-gnueabihf
Configured with: ../src/configure -v --with-pkgversion='Raspbian 8.3.0-6+rpi1' --with-bugurl=file:///usr/share/doc/gcc-8/README.Bugs --enable-languages=c,ada,c++,go,d,fortran,objc,obj-c++ --prefix=/usr --with-gcc-major-version-only --program-suffix=-8 --program-prefix=arm-linux-gnueabihf- --enable-shared --enable-linker-build-id --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --libdir=/usr/lib --enable-nls --enable-bootstrap --enable-clocale=gnu --enable-libstdcxx-debug --enable-libstdcxx-time=yes --with-default-libstdcxx-abi=new --enable-gnu-unique-object --disable-libitm --disable-libquadmath --disable-libquadmath-support --enable-plugin --with-system-zlib --with-target-system-zlib --enable-objc-gc=auto --enable-multiarch --disable-sjlj-exceptions --with-arch=armv6 --with-fpu=vfp --with-float=hard --disable-werror --enable-checking=release --build=arm-linux-gnueabihf --host=arm-linux-gnueabihf --target=arm-linux-gnueabihf
Thread model: posix
gcc version 8.3.0 (Raspbian 8.3.0-6+rpi1)
```

**Comment by jarun** [comment]
Fixed at ccce80a1e91b99c58d1a4e1453b5238036e84f22.

Will make a new release soon.

**Comment by jungle-boogie** [comment]
Verified fixed with git commit 29ef23230197faf2b125b9abb8e16f14a7f83769
